source/_drafts/Introducing-Video-js-HTTP-Streaming-VHS.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Introducing Video.js HTTP Streaming (VHS)
+author:
+  name: Garrett Singer
+  github: gesinger
+tags:
+  - vhs
+---
+
+"How do I get my video to play with Video.js?"
```

Comment:
might want to make this a [pullquote](https://hexo.io/docs/tag-plugins#Pull-Quote)

Reply:
I gave it a shot, but it looked weird when rendered on the site.

Reply:
You may want to just use `>` for this rather than pullquote directly.